collector/manager/metrics_collector_manager.go

```diff
@@ -211,9 +224,9 @@ func (mcm *MetricsCollectorManager) declareMetricDefinitions(metricDetails []col
 
 		// put all the tags together for the full list of tags to be applied to this metric definition
 		allMetricTags := tags.Tags{}
-		allMetricTags.AppendTags(globalTags)   // global tags are overridden by...
+		allMetricTags.AppendTags(metricTags)   // metric tags are overriden by
 		allMetricTags.AppendTags(endpointTags) // endpoint tags which are overridden by...
-		allMetricTags.AppendTags(metricTags)   // metric tags.
+		allMetricTags.AppendTags(globalTags)   // global tags
```

Comment:
The metric tags are set by a user of a project.
The global tags are set by the admin of the cluster.

A user should not be able to overwrite what an admin has decided should be stored for a tag. They should only have the ability to add tags, not modify them.